website/docs/patterns/table-multi-select/partials/specifications/specifications.md

```diff
@@ -0,0 +1,49 @@
+## Anatomy
+
+![Multi-select anatomy](/assets/patterns/table-multi-select/selection-anatomy.png)
+
+| Element | Usage | 
+|---------|-------|
+| Selected count | Displays the number of selected items in the table. |
+| Selected message | Communicates that results in the table have been selected. |
+| Total results  | Displays the total number of results in the data set. |
+| Actions trigger | Recommended to use either a `secondary` or `critical` [Button](/components/button)for single actions, or a `secondary` [Dropdown](/components/dropdown) for multiple actions. |
+| Bulk actions | Consolidates multiple bulk actions into a single element. |
+| Bulk selection | Consolidates multiple selection functions into a single element. |
+
+## Spacing
+
+### Multi-select pattern
+
+Within the Multi-select pattern itself, use a 16px horizontal gap between the message and the actions.
+
+![Spacing within the multi-select pattern](/assets/patterns/table-multi-select/multi-select-pattern-spacing.png)
+
+### In context
+
+When combining a Multi-select pattern with filtering and similar functional elements around the Table, using a 16px vertical gap be filter elements, Multi-select pattern, and Table.
```

Comment:
```suggestion
When combining a Multi-select pattern with filtering and similar functions acting on Table data, use a 16px vertical gap between rows of related elements.
```

Reply:
Tried to clarify this and fix grammar. May still want to tweak a bit if I misunderstood the intent.

Reply:
Nope, that sums it up more cleanly, thank you!